frontend/src/components/elements/BuildingDetailElement/EditBuildingPopup.tsx

```diff
@@ -153,6 +173,37 @@ export default function EditBuildingPopup({open, setOpen, prevName, prevAddress,
                                 )}
                             />
                         </FormControl>
+                        <div className={styles.field}>
+                            <Button
+                                variant="contained"
+                                component="label"
+                                sx={{'width': 240, 'fontSize': 12, 'backgroundColor': 'var(--primary-dark)',
+                                    '&:hover': {
+                                        backgroundColor: 'var(--secondary-dark)',
+                                    }}}
+                            >
+                                Handleiding (PDF)
+                                <input
+                                    type="file"
+                                    onChange={(e) => setFormPDFGuide(e.target.files ? e.target.files[0] : null)}
+                                    accept="application/pdf"
+                                    hidden
+                                />
+                            </Button>
+                            <IconButton onClick={() => setFormPDFGuide(null)}>
+                                <CloseIcon/>
+                            </IconButton>
+                        </div>
+                        <p style={{fontSize: 14}} className={styles.field}>
+                            {
+                                formPDFGuide ?
+                                    (formPDFGuide.name.length > 40 ?
+                                        formPDFGuide.name.slice(0, 37) +'...' :
```

Comment:
Could it be possible to use a `<Typography noWrap>` element? The `noWrap` prop does this:

> If true, the text will not wrap, but instead will truncate with a text overflow ellipsis

[See docs](https://mui.com/material-ui/api/typography/#props)

Reply:
I tried it, but it doesn't seem to work for whatever reason. Seems like a small issue to waste time on at this point though